microshift_release_notes/microshift-4-12-release-notes.adoc

```diff
@@ -1,13 +1,23 @@
+:_content-type: ASSEMBLY
 [id="microshift-4-12-release-notes"]
 = {product-title} {product-version} release notes
 include::_attributes/attributes-microshift.adoc[]
 :context: release-notes
 
-Do not add or edit release notes here. Edit release notes directly in the branch
-that they are relevant for.
+toc::[]
 
-Release note changes should be added/edited in their own PR.
+// Update All for MicroShift; text is FPO (for placement only)
+Red Hat {product-title} provides developers and IT organizations with an OpenShift-like experience for small form factor and edge computing, delivered as an application that customers can deploy on top of their managed RHEL devices at the edge. {product-title} is designed to make control plane restarts economical and be lifecycle-managed as a single unit by the operating system. Updates, roll-backs, and config changes become a matter of staging another version in parallel and then - without relying on a network - flipping to/from that version and restarting.
```

Comment:
```suggestion
Red Hat {product-title} provides developers and IT organizations with an OpenShift-like experience for small form factor and edge computing, delivered as an application that customers can deploy on top of their managed RHEL devices at the edge. {product-title} is designed to make control plane restarts economical and be lifecycle-managed as a single unit by the operating system. Updates, roll-backs, and configuration changes become a matter of staging another version in parallel and then - without relying on a network - flipping to or from that version and restarting.
```
to/from: see https://www.ibm.com/docs/en/ibm-style?topic=punctuation-slashes
config: see https://www.ibm.com/docs/en/ibm-style?topic=grammar-abbreviations

I am not sure about, and cannot find any guidance on using 'OpenShift-like'.

Reply:
@ShaunaDiaz  for the preview, can you generate one without the slashes so we can see what the rendering is (and then add them back per your needs)? Thanks!

Reply:
/remove-label peer-review-in-progress
/label peer-review-done